cmd/calico-node/main.go

```diff
@@ -40,7 +40,8 @@ var flagSet = flag.NewFlagSet("Calico", flag.ContinueOnError)
 // Build the set of supported flags.
 var version = flagSet.Bool("v", false, "Display version")
 var runFelix = flagSet.Bool("felix", false, "Run Felix")
-var runStartup = flagSet.Bool("startup", false, "Initialize a new node")
+var runStartup = flagSet.Bool("startup", false, "Initialize a new node with constant monitoring of IP address")
```

Comment:
Sorta a nit, but I think it's fair to say that these are really separate options.

I'd recommend

```
flagSet.Bool("startup", false, "Initialize a new node")
flagSet.Bool("monitor-addresses", "false", "Monitor for node IP address changes")
```

Reply:
okie.